### ISSUE TITLE
Unable to configure deadpendency correctly for Celery

### ISSUE THREAD

**Issue by thedrow** [deadpendency/deadpendency#52]
## Expected Behavior

Deadpendency should analyze all dependencies.

## Current Behavior
Analysis is invalid without a hint to why.

## Example Of Issue / Steps To Reproduce
See PR https://github.com/celery/celery/pull/7242.

## Other Comments
At the very least, I'd like the UI to tell me what to fix. Is this a schema issue? Is this because of invalid YAML syntax?
As things stand now, I can't use it without your guidance. Are globs supported (asides from the default)?

**Comment by AlistairB** [comment]
My apologies, I agree this was not clear. For now I have made this explicit in https://deadpendency.com/docs/config#additional-dependency-files .

Ideally this would also provide an error stating globbing is not supported as a fast fail. I have created https://github.com/deadpendency/deadpendency/issues/54 to this purpose.

I can see how this is cumbersome as well, so I have created https://github.com/deadpendency/deadpendency/issues/53 to add wildcard support. Full globbing might be challenging as there can be rate limiting problems if Deadpendency needs to search through every directory in the repository.

Anyway, for now you will be forced to specify each file as a separate `additional-dependency-files` entry with a full path to the file. Sorry about that. I'm going to track resolution in the other issues. They should be done within 2 weeks. Thanks!
